spec/classes/mod/jk_spec.rb

```diff
@@ -12,13 +12,19 @@
     it { is_expected.to contain_file('jk.conf').that_notifies('Class[apache::service]') }
   end
 
-  context "RHEL 6 with only required facts and no parameters" do
+  default_ip = '192.168.1.1'
```

Comment:
I had a problem in #1671 regarding this change. I'll wait 'till we solve that then replicate the solution here.